FirebaseInstallations/Source/Library/Public/FirebaseInstallations/FIRInstallations.h

```diff
@@ -22,9 +22,11 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /** A notification with this name is sent each time an installation is created or deleted. */
-FOUNDATION_EXPORT const NSNotificationName FIRInstallationIDDidChangeNotification;
+FOUNDATION_EXPORT const NSNotificationName FIRInstallationIDDidChangeNotification
+    NS_SWIFT_NAME(InstallationIDDidChange);
```

Comment:
The `Notification` was already dropped here in Swift, the only difference is a lack of `FIR` prefix now.